ValidBST.py

```diff
@@ -0,0 +1,34 @@
+# TC : O(N)
+# SC : O(N)
```

Comment:
For DFS SC will be O(H). Height of the tree

Reply:
Plus for loop is having O(N) so you can say like N+H

Reply:
But if you check the conditions for value is lesser than its previous element while doing inorder(Tree Traversal) then you don't need to store the tree nodes in list